jobserver/authorization/roles.py

```diff
@@ -38,6 +38,17 @@ class StaffAreaAdministrator:
     ]
 
 
+class ServiceAdministrator:
+    display_name = "Service Administrator"
+    description = """Access the Create Project page.
```

Comment:
It would be worth adding something like "DO NOT USE", or similar, just in case someone is tempted to assign someone to this role. It shouldn't matter right now, because there are no permissions associated with it, but there will be shortly.